src/packages/scripts/saveAdConversion/saveAdConversion.ts

```diff
@@ -0,0 +1,27 @@
+import { WebflowScript } from '../../types'
+import { getAdClickBrowserContext, getAdConversionFromHref } from './helper'
+
+const saveAdConversion: WebflowScript = {
+  requireFeatureFlag: 'webflow_script_save_ad_conversion',
+  handler: () => {
+    const { isConversion, ...adConversionParams } = getAdConversionFromHref()
+    if (isConversion) {
+      const adClickBrowserContext = getAdClickBrowserContext()
+      const adConversion = {
+        ...adConversionParams,
+        ...adClickBrowserContext,
+      }
```

Comment:
similar to my comment above, consider a naming similar to
```suggestion
    const { isConversion, ...adConversionParams } = getAdConversionFromHref()
    if (isConversion) {
      const browserContext = getBrowserContext()
      const adConversionBrowserContext = {
        ...adConversionParams,
        ...browserContext,
      }
```